### ISSUE TITLE
Graph Editor: Mouse UI action orbits camera in render view

### ISSUE THREAD

**Issue by HardCoreCodin** [AcademySoftwareFoundation/MaterialX#1984]
1. When the scrollbar appears in the property editor, grabbing it with the mouse to scroll and dragging the scrollbar up next to the render view, treats it as a mouse drag in the render view and orbits the camera.
2. Using the File menu and clicking a menu option and dragging the mouse also does the same and orbits the camera as it is over the render view.

**Comment by kwokcb** [comment]
There are bunch of UI items [here](https://github.com/AcademySoftwareFoundation/MaterialX/issues/1398). I think your case  is similar to another item mentioned.

**Comment by LeaRe** [comment]
Hi! I would like to tackle this issue for the ASWF Dev Days 2024.

**Comment by LeaRe** [comment]
While working on this issue I also noticed another case of unintentional camera orbiting. The previous logic to detect the cursor in the render view did not consider that the render view moves up when scrolling. As a result, the camera would
still orbit when clicking parts of the Node Property Editor that were previously occupied by the render view. I added a fix for this behaviour as well in my changes for this ticket (PR will be out soon).
